### ISSUE TITLE
Clickable elements should be buttons, or at least have a tab index (usability/accessibility issue)

### ISSUE THREAD

**Issue by superuntitled** [contentco/quill-emoji#154]
The span.ap elements in the **#tab-panel** and the **span.textarea-emoji-control** element should be **button** elements and not **span** elements. 

This would allow for using the tab key to cycle through the emojis, and not rely on using the mouse (browsers give buttons a tabindex and can be triggered using the spacebar or return key).
